src/js/player.js

```diff
@@ -26,19 +26,19 @@ import AudioTrackList from './tracks/audio-track-list.js';
 import VideoTrackList from './tracks/video-track-list.js';
 
 // Include required child components (importing also registers them)
-import MediaLoader from './tech/loader.js';
-import PosterImage from './poster-image.js';
-import TextTrackDisplay from './tracks/text-track-display.js';
-import LoadingSpinner from './loading-spinner.js';
-import BigPlayButton from './big-play-button.js';
-import ControlBar from './control-bar/control-bar.js';
-import ErrorDisplay from './error-display.js';
-import TextTrackSettings from './tracks/text-track-settings.js';
+import './tech/loader.js';
```

Comment:
I would much prefer the earlier version. It's much more explicit and reduces cognitive load. I don't have to go to `./tracks/text-track-settings.js` to *know* that in here I can refer to it via `TextTrackSettings`.